.appveyor.yml

```diff
@@ -3,16 +3,16 @@ clone_depth: 10
 version: "{build}"
 
 environment:
+  fast_finish: true
   matrix:
-    - nodejs_version: "6.9.1"
-      platform: x86
-    - nodejs_version: "7"
+    - nodejs_version: "6"
```

Comment:
Ah, OK, that's reasonable. Maybe have a comment for the future in this file?

I'm not affiliated with AppVeyor but I guess you could report any issues to them :)
On a side note, did you compare cold phase? Because caching isn't enabled if the build fails.